fuzzing/read_lgl_fuzzer.c

```diff
@@ -41,7 +43,7 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size){
 	FILE *ifile;
 	ifile = fopen("/tmp/libfuzzer.gml", "r");
 	if(ifile == 0){
-		unlink(filename);
```

Comment:
What's wrong with `unlink()`? Is it not supported with AFL++?

Also, I'm not familiar with `std::__fs` but the double-underscore seems to imply to me that this is an implementation detail. If we have to use `std::` stuff, can't we get away with `std::filesystem::remove_all()` instead?

Reply:
We are compiling the file as C++ instead of as C and `unlink()` is therefore not supported.

> can't we get away with `std::filesystem::remove_all()` instead?

Unfortunately not. Tested locally just now.

Reply:
Okay, how about [`remove()`](https://en.cppreference.com/w/c/io/remove) then? That one is supposed to be portable since C89/C90 and is equivalent to `unlink()`. Sorry for nitpicking, I'm just generally uneasy about using internal stuff.

Reply:
remove() works fine locally. It has been added!